src/App.js

```diff
@@ -37,7 +38,9 @@ const tweetsArray = [
 function App() {
   return (
```

Comment:
Buenisimo Rod pero imagina que en el array tenemos 100 elementos esto ya no seria rentable para eso podemos hacer uso de un map y pasar el index para el prop key y de esta manera eliminar el warning recuerda que este key va dentro de la tag map 'padre' dentro del map 😉 
```suggestion
  return (
    <div className="App">
      {tweetsArray.map((Onetweet, index) => (
        <Tweet key={index} tweet={Onetweet} />
      ))}
    </div>
  );
```